nncf/onnx/graph/metatypes/onnx_metatypes.py

```diff
@@ -711,12 +745,9 @@ def _is_depthwise_conv(model: onnx.ModelProto, node: onnx.NodeProto) -> bool:
     :param node: Convolution node to check whether it is depthwise.
     :return: True if the convolution is depthwise, False - otherwise.
     """
-    conv_group = None
```

Comment:
After changes `conv_group` used while definition is under if condition, as result possible that `conv_group` uses without definition and raise error.

Reply:
Since **ONNXDepthwiseConvolutionMetatype** became the subtype of the **ONNXGroupConvolutionMetatype** that must have a _group_ attribute, it's not possible that _conv_group_ wouldn't be found.